plugins/modules/aci_access_port_to_interface_policy_leaf_profile.py

```diff
@@ -351,12 +385,13 @@ def main():
         ))
 
     aci = ACIModule(module)
+    class_rn = 'infraFexP/fexprof' if type_profile == 'fex' else 'infraAccPortP/accportprof'
```

Comment:
Define both class and rn into separate variable, there is no need to define one then split it.